models/Asset/Document/ImageThumbnail.php

```diff
@@ -57,14 +57,25 @@ public function __construct($asset, $config = null, $page = 1, $deferred = true)
     }
 
     /**
-     * @param bool $deferredAllowed
+     * TODO: Pimcore 11: Change method signature to getPath($args = [])
+     *
+     * @param mixed $args,...
      *
      * @return string
      */
-    public function getPath($deferredAllowed = true)
+    public function getPath(...$args)
     {
+        // TODO: Pimcore 11: remove calling the covertArgsBcLayer() method
+        $args = $this->convertArgsBcLayer($args);
+
+        // set defaults
+        $deferredAllowed = $args['deferredAllowed'] ?? true;
+        $forceFrontend = $args['forceFrontend'] ?? false;
```

Comment:
but that change from `forceFrontend` to `frontend` was on purpose to suggest a rename of this "parameter" key name as well, because it is IMHO not self-explainatory of what is forcing and/or would be hard to remember all the rules behind it.
While using `frontend`, knowing that the rule is that only in frontend this is by default true, then all would go along, one can set what one wants to get accordingly.


OT: noticed you are applying the same changes to the PR for 11.x branch, but JFYI that's not necessary overhead, we usually merge PR in 10.6, then we merge all 10.6 into 11.x, so in https://github.com/pimcore/pimcore/pull/14618 you would have just to rebase it and work on top of the changes former 10.6 but just have to wait the merges.